src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -379,13 +379,18 @@ WTF::String ERR_INVALID_ARG_TYPE(JSC::ThrowScope& scope, JSC::JSGlobalObject* gl
     unsigned length = expected_types.size();
     if (length == 1) {
         result.append(expected_types.at(0).toWTFString(globalObject));
+    } else if (length == 2) {
+        result.append(expected_types.at(0).toWTFString(globalObject));
+        result.append(" or "_s);
+        result.append(expected_types.at(1).toWTFString(globalObject));
     } else {
         for (unsigned i = 0; i < length - 1; i++) {
             JSValue expected_type = expected_types.at(i);
-            if (i > 0) result.append(", "_s);
             result.append(expected_type.toWTFString(globalObject));
+            RETURN_IF_EXCEPTION(scope, {});
```

Comment:
is this for expected_types.at() or expected_type.toWTFString()? and if it's for toWTFString, shouldn't the other result.append()s have it too?

Reply:
`expected_type.toWTFString()`. but this is an internal function and its illegal for `expected_type` to not be a string so ill remove